includes/class-sensei-wc.php

```diff
@@ -1333,7 +1337,7 @@ public static function cancel_order( $order_id ) {
 		if ( 0 < sizeof( $order->get_items() ) ) {
 
 			// Get order user
-			$user_id = $order->__get( 'user_id' );
+			$user_id = $order->get_customer_id();
```

Comment:
While I suspect that this change is a good change, we should try to keep commits as granular as possible. This also applies to the scope of a PR, so for example I'd remove this change and perhaps create a new PR that has multiple of these fixes in it. 
We have a bit more information on this here: https://vip.wordpress.com/documentation/creating-good-changesets/

Also, this is a small change but while in most cases I think being more verbose is good, in this case I would remove line 1239 and replace 1241 by something like:
`if ( ! in_array( $order_status,  apply_filters( 'sensei_wc_add_order_stati', array( 'wc-completed', 'wc-processing' ) ) ) ) {`
(I say something like, because I haven't actually tested it)

The reasoning for this is that in this case it's making the smallest change to the current code and that the $accepted_order_stati  and $additional_order_stati are never used again.

In another not very important feedback is that if we were to keep with the separate variables I would remove the blank line L1238 since those 2 lines should be logically grouped together. blank lines should be used to demonstrate some sort of logical separation :)

Reply:
Thanks, @sboisvert, I'll get make the changes